.eslintrc

```diff
@@ -31,6 +31,7 @@
   "rules": {
     "arrow-parens": 2,
     "block-scoped-var": 0,
+    "brace-style": ["error", "1tbs", {"allowSingleLine": false}],
```

Comment:
I would prefer to use `2` instead of `"error"`, here and on the new `"object-property-newline"` eslint rules configuration, for consistency with the rest of the rules.